lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -631,12 +651,22 @@ impl LayerModel {
         self.sublayers.borrow().all()
     }
 
-    fn add_sublayer(&self, layer: &Layer) {
+    /// Attach a `layer` as a sublayer.
+    pub fn add_sublayer(&self, layer: &Layer) {
         let ix = self.sublayers.borrow_mut().layers.insert(layer.downgrade());
         self.sublayers.borrow_mut().layer_placement.insert(layer.id(), ix);
         layer.add_parent(&self.sublayers);
     }
 
+    /// Remove previously attached sublayer.
+    ///
+    /// The implementation is a reverse of [`LayerModel::add_sublayer`]: we modify both fields of
```

Comment:
I believe that "The implementation is a reverse" is not a valid english. Please check it with native speaker.